### PR TITLE
Filtering fp of dynatrace behavior

### PR DESCRIPTION
**Pull request by redsand**
Example fp:

```
2022-02-09 15:29:35 example.host.com Sysmon: 1: Process Create | RuleName=technique_id=T1059,technique_name=Command-Line Interface | UtcTime=2022-02-09 15:29:35.190 | ProcessGuid={91e7d9e6-dddf-6203-cb2a-010000003d00} | ProcessId=8204 | Image=C:\Windows\System32\cmd.exe | FileVersion=10.0.17763.1697 (WinBuild.160101.0800) | Description=Windows Command Processor | Company=Microsoft Corporation | OriginalFileName=Cmd.Exe | OriginalCommandLine="C:\Windows\system32\cmd.exe" /C ICACLS "C:\ProgramData\dynatrace\gateway\config\connectivity.history" /reset | CommandLine=C:\Windows\system32\cmd.exe /C ICACLS C:\ProgramData\dynatrace\gateway\config\connectivity.history /reset | CurrentDirectory=C:\Windows\system32\ | User=NT AUTHORITY\SYSTEM | LogonGuid={91e7d9e6-71e9-61f8-e703-000000000000} | LogonId=0x3e7 | TerminalSessionId=0 | IntegrityLevel=System | Hashes=SHA1=DED8FD7F36417F66EB6ADA10E0C0D7C0022986E9,MD5=911D039E71583A07320B32BDE22F8E22,SHA256=BC866CFCDDA37E24DC2634DC282C7A0E6F55209DA17A8FA105B07414C0E7C527,IMPHASH=272245E2988E1E430500B852C4FB5E18 | ParentProcessGuid={91e7d9e6-dda8-6203-872a-010000003d00} | ParentProcessId=16408 | ParentImage=C:\Windows\System32\msiexec.exe | OriginalParentCommandLine=C:\Windows\System32\MsiExec.exe -Embedding 37816432465B18C15156CD3456A6D41C E Global\MSI0000 | ParentCommandLine=C:\Windows\System32\MsiExec.exe -Embedding 37816432465B18C15156CD3456A6D41C E Global\MSI0000 | ParentCommandLineObfuscated=false | pid=15900 | level=0 | sys_version=5 | category=Process Create (rule: ProcessCreate) | op=Info | Microsoft-Windows-Sysmon/Operational=true | key=0x8000000000000000 | id=5848858 | app="C:\Windows\sysmon64.exe" | tid=14172 | channel="Microsoft-Windows-Sysmon/Operational" | pid_user="NT AUTHORITY\SYSTEM" | sid=S-1-5-18 | account type=User | type=notice(4) 
```

### CODE REVIEW

File: rules/windows/process_creation/win_file_permission_modifications.yml [hunk 1]
```diff
@@ -19,14 +19,21 @@ detection:
       CommandLine|contains: '/grant'
     - Image|endswith: '\attrib.exe'
       CommandLine|contains: '-r'
-  condition: selection
+  filter:
+    - CommandLine: 
+        - 'C:\Windows\system32\cmd.exe /C ICACLS C:\ProgramData\dynatrace\gateway\config\config.properties /grant :r *S-1-5-19:F'
```
Comment (Neo23x0) [inline rules/windows/process_creation/win_file_permission_modifications.yml:24]: Does that `*` character really appear in the command line or do you want to use a wildcard. 
If it's the latter, please use `CommandLine|contains|all` instead and split that string into two.

**Comment by frack113** [comment]
proposal :
```
  filter_reset:
    CommandLine|endswith: 'ICACLS C:\ProgramData\dynatrace\gateway\config\connectivity.history /reset'
  filter_grant:
    CommandLine|contains|all:
      - 'ICACLS C:\ProgramData\dynatrace\gateway\config\config.properties /grant :r '
      - 'S-1-5-19:F'
```

**Comment by redsand** [comment]
this has been completed

**Comment by redsand** [comment]
any thing else on this i need to address? or can this be merged?

**Pull request merged**